### ISSUE TITLE
Fix existing tuner input type to display all options and configure, it currently does not

### ISSUE THREAD

**Issue by eelpout** [google/sagetv#473]
Currently, with v9 (as of 01/2022) when one clicks on an _existing_ tuner to change input type (i.e. Component+SPDIF, Composite etc.) it doesn't display _all_ options that are shown when adding a _new_ tuner (i.e. Component with analog RCA audio input is missing). 

This would be helpful to flush this out and display every option. One example case is when there are occasional issues with HD-PVR and digital audio sources and one needs as easy way to alter the input without having to delete and then re-add the tuner.

I'm not sure if this is a problem with the way HD-PVR tuner type is defined or a more general problem with all tuners.

**Comment by JustFred51** [comment]
If memory serves me correctly, the behavior described may occur if you had added the HDPVR device prior to SageTv v 9.1.8, or had manually edited the HDPVR's .inf file (per an old recommendation in the Sage forums).  Does this correspond to the conditions of your installation?

**Comment by eelpout** [comment]
yep, very old original install (2006???). Not sure exactly when the first HD-PVR's were added, but _way_ before v9.

Am I understanding correctly that if I was to remove these tuners and re-add post v9.1.8 I would get the behavior I'd expect?

**Comment by JustFred51** [comment]
Not sure exactly which version of Sage you're running, but there's no need to re-install Sage as long as you're running version v9.1.8 or newer.  I haven't walked thru your exact scenario, but my recommendation is as follows:
1. Was the .inf file for the HDPVR manually edited (per an old recommendation from the Sage forums)?  If so, use Device Manager to uninstall the driver and then re-install it (with Sage stopped).  Relaunch Sage and see if the issue is resolved.  If not, continue below.
2. Stop Sage and save your existing sage.properties file (just in case...).  Manually edit sage.properties to remove all mmc/encoders lines associated with the HDPVR.  Restart Sage and re-install the capture device in Sage.

**Comment by eelpout** [comment]
I'm on v9.2.5.

I haven't edited the .inf files that I can remember, but in 15 years who knows. ;)

I have tweaked the sage.prop and the mmc entries for various things like ExeMultiTuner IP settings (DirecTV tuning) and recording volume destination, but nothing else.

OK, I'll give your steps a try in a few days and see what happens. Thx.
